portfolio/src/main/webapp/index.html

```diff
@@ -441,11 +441,11 @@ <h4>Justice for Ahmaud Arbery</h4>
                   <hr />
                   <h4>How to Help</h4>
                   <p>
-                    <b>Donate Here: </b
-                    ><a class="interactive-pics" href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a
+                    <b>Donate Here: </b>
+                    <a class="interactive-pics" href="https://www.gofundme.com/f/i-run-with-maud">I Run With Maud</a
                     ><br />
-                    <b>Sign Petitions Here: </b
-                    ><a
+                    <b>Sign Petitions Here: </b>
+                    <a
                       class="interactive-pics"
                       href="https://www.change.org/p/liberty-county-distric-attorney-justice-for-ahmaud-arbery"
                       >Justice for Ahmaud Arbery</a
```

Comment:
nit: I think this format should be consistent. So lines 508 and 509 should look like this too. Same thing for 694.

Reply:
I updated make pretty so the formatting is more consistent now.